config/initializers/elasticsearch.rb

```diff
@@ -56,8 +69,8 @@ def es_config_non_production
 end
 
 config = ELASTIC_CONFIG['vcap_services'].present? ?
-            es_config_production :
-            es_config_non_production
+            es_config_from_vcap :
```

Comment:
It's now not clear if you are connection to production elasticsearch or locally

Reply:
That's the question I'm asking in the original comment above. Why not support VCAP_SERVICES locally? This would be useful when you want to populate a PaaS ES index from your local machine. We need to discuss generally about Rails's environments and our own environments: we may run in production mode on our staging server, etc.

Reply:
Moreover, I don't think there should be code that's environment specific anywhere in the application. It opens the door to things behaving differently depending on what environment it's in. Code coverage and testing also become difficult as you have tests that also depend on the environment.